### PR TITLE
Update readme and swagger

### PR DESCRIPTION
**Pull request by kolya182**
New Swagger look http://localhost:8423/swagger/index.html
Screen Shot 2020-03-03 at 5 13 57 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,22 +15,22 @@ The main features of it are:
...
 1. Spin up a Redis instance: `docker run -it -p 6379:6379 redis`
-1. Start the app: `make start`
+2. Start the app: `make start`
    1. Explore the API: [http://localhost:8423/swagger/index.html](http://localhost:8423/swagger/index.html)
    1. Use the API:
       * `curl -v "http://localhost:8421/v1/market/info?coin=60" | jq .`
       * `curl -v -X POST 'http://localhost:8421/v1/market/ticker' -H 'Content-Type: application/json' -d '{"currency":"ETH","assets":[{"type":"coin","coin":60}]}'`
       * `curl -v "http://localhost:8421/v1/market/charts?coin=60&time_start=1574483028" | jq .`
-1. When done: `make stop`
+3. When done: `make stop`
```
Comment (arein) [inline README.md]: nitpick: GitHub's Markdown flavor automatically increases the rendering of numbered lists. So what we had before I argue was better

Or is this also rendered somewhere else?

File: go.mod [hunk 2]
```diff
@@ -5,14 +5,20 @@ go 1.13
...
+	github.com/mailru/easyjson v0.7.1 // indirect
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/spf13/viper v1.6.2
 	github.com/stretchr/testify v1.5.1
 	github.com/swaggo/gin-swagger v1.2.0
 	github.com/swaggo/swag v1.6.5
 	github.com/trustwallet/blockatlas v1.0.37
 	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5 // indirect
+	golang.org/x/net v0.0.0-20200301022130-244492dfa37a // indirect
+	golang.org/x/tools v0.0.0-20200303225724-c5a141475315 // indirect
```
Comment (arein) [inline go.mod]: it's not clear what these new dependencies are needed for. Mind explaining?

Comment (kolya182) [reply go.mod]: Populated after `make install`, not sure why

File: README.md (hunk 1 above)
Comment (kolya182) [reply README.md]: Did not know, but this way steps numeration sequence more explicite, instruction for humans not for machine to read.

Comment (arein) [reply README.md]: Don't feel that strong on this but IMO it's better to just use `1.` so you don't need to maintain the order

File: go.mod (hunk 2 above)
Comment (arein) [reply go.mod]: Sounds like we don't need them though?

Comment (kolya182) [reply go.mod]: Removed, not sure why go does it.

Comment (kolya182) [reply go.mod]: ok

**Comment by arein** [comment]
Sorry @kolya182 just squashed and merged and now it says I did the change. Won't happen again!

**Comment by kolya182** [comment]
Looking good

**Pull request closed without merging**